config/release/release.py

```diff
@@ -71,7 +71,7 @@ def __query( url ) :
 os.makedirs( buildDir )
 
 dependenciesReleases = __query( "https://api.github.com/repos/GafferHQ/dependencies/releases" )
-dependenciesRelease = next( r for r in dependenciesReleases if len( r["assets"] ) )
+dependenciesRelease = next( r for r in dependenciesReleases if len( r["assets"] ) and r["tag_name"] <= args.tag )
```

Comment:
Assuming `args.tag` and `r["tag_name"]` are strings, I'm not sure this test is accurate:

```
>>> '0.36.5.0' < '0.36.10.0'
False
```

Maybe it'll do for our simple case of dependencyReleases all being 0.*.0.0 though?